src/renderer/components/input/input.validators.ts

```diff
@@ -44,7 +44,7 @@ export const isUrl: Validator = {
 
 export const isPath: Validator = {
   condition: ({ type }) => type === "text",
-  message: () => _i18n._(t`This field must be a path to an existing file`),
+  message: () => _i18n._(t`This field must be a path to an existing folder`),
```

Comment:
"file" I think is more accurate (especially on *nix systems).

This really should be two different validators: `isFilePath` and `isFolderPath`

Reply:
I agree, should be two different validators but that's out of scope of this PR.